tests/acceptance/chartcuterie/test_image_block_builder.py

```diff
@@ -115,3 +116,10 @@ def test_image_block_for_function_regression(self):
 
         assert image_block and "type" in image_block and image_block["type"] == "image"
         assert "_media/" in image_block["image_url"]
+
+    def test_image_not_generated_without_support(self):
```

Comment:
And therefore if it wasn't part of the supported issues it would throw an error so adding unsupported issue here to the test. https://sentry.sentry.io/issues/5434963970/